packages/glimmer-syntax/lib/parser.ts

```diff
@@ -19,7 +19,7 @@ export const syntax = {
   Walker
 };
 
-export function preprocess(html, options?) {
+export function preprocess(html, options?): Object {
```

Comment:
I think we should either leave out the return type or type it properly, putting `Object` there is quit misleading imo